tests-functional/tests/test_logging.py

```diff
@@ -0,0 +1,63 @@
+import re
+import time
+from test_cases import StatusBackend
+import pytest
+import os
+
+@pytest.mark.rpc
+@pytest.mark.skip("waiting for status-backend to be executed on the same host/container")
+class TestLogging:
+
+    @pytest.mark.init
+    def test_logging(self, tmp_path):
+        await_signals = [
+            "mediaserver.started",
+            "node.started",
+            "node.ready",
+            "node.login",
+        ]
+
+        backend_client = StatusBackend(await_signals)
+        assert backend_client is not None
+
+        # Init and login
+        backend_client.init_status_backend(data_dir=str(tmp_path))
+        backend_client.create_account_and_login(data_dir=str(tmp_path))
+        key_uid = self.ensure_logged_in(backend_client)
+
+        # Configure logging
+        backend_client.rpc_valid_request("wakuext_setLogLevel", [{"logLevel": "ERROR"}])
+        backend_client.rpc_valid_request("wakuext_setLogNamespaces", [{"logNamespaces": "test1.test2:debug,test1.test2.test3:info"}])
+
+        # Re-login (logging settings take effect after re-login)
```

Comment:
hmm, why? I though there's an atomic setting now which applies right away 🤔

Reply:
Yes, it is technically possible to set up logging at runtime. I followed the flow of `setLogLevel` for now, i.e., setLogLevel -> save settings in the database -> relogin -> load settings from database -> set up logging.

I'll address it in a follow-up to make it work at runtime and remove the popup about app restart requirement on desktop.